### DISCUSSION TITLE
Cannot get internet on router

### DISCUSSION THREAD

**Discussion by derek-teay** [SmoothWAN/SmoothWAN#185] [General & Help]
Hi all, I have a Slate AX that I installed SmoothWAN on and got Speedify paid for and setup. I have 2 WANs, normal ISP and a T-Mobile cellular backup, they are plugged into the correct ports and I can see them in Speedify. 

In the free LAN port, I have that running into my TP-Link Deco router, which says it has internet. However, when I try to connect to the Deco router, any device doesn't seem to have internet access. I suspect this is a DHCP problem and I'm just not too sure how to fix it on the Slate AX. I think the best option would be to just "bridge" the Slate AX and let the router handle DHCP (unless you think it's better to do the opposite) but how can I do that? I searched and didn't find anything that helped. Thanks!

**Comment by bt61** [comment]
https://www.cyberciti.biz/humour/a-haiku-about-dns/

**Comment by TalalMash** [comment]
Yes the correct way is to bridge unless you need specific TP-link features, or as @bt61 suggested you may need to point the DNS on the Deco to the gateway IP 172.17.17.2 , client IP should be static as well, preferable under 172.17.17.10
